src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -19,14 +19,15 @@
 
 package com.spotify.docker.client;
 
-import com.google.common.base.Optional;
-import com.google.common.base.Preconditions;
-import com.google.common.io.CharStreams;
-import com.google.common.net.HostAndPort;
+
 import com.fasterxml.jackson.core.JsonGenerator;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.google.common.base.Optional;
```

Comment:
Sorry for the nitpick, but could we avoid reordering imports here? It makes the diff misleading.